### DISCUSSION TITLE
What happened to Managed Peripheral Support?

### DISCUSSION THREAD

**Discussion by jaredballen** [shinyorg/shiny#1219] [Q&A] [answered]
It appears that support for managed peripheral was removed on April 1st. the commit comment seemed to indicate this was part of a larger refactor. Are there plans on adding this back in?

**Comment by aritchie** [comment, marked as answer]
No.  There are no plans to add it back.  It isn't needed anymore anyhow

**Reply by jaredballen** [reply]
Got it. Thanks @aritchie!
